docs/static/sass/styles.scss

```diff
@@ -6,10 +6,6 @@
 @import 'codehilite';
 @include codehilite;
 
-td, th {
-  padding: 1rem 0.75rem;
-}
-
```

Comment:
Why do we prefer inline styles for the padding? Wouldn't it be better to just have some default padding?